qa/workunits/suites/ffsb.patch

```diff
@@ -1,7 +1,41 @@
-diff -urp 1/parser.c 2/parser.c
---- 1/parser.c	2008-10-28 04:17:05.000000000 +0800
-+++ 2/parser.c	2018-06-26 20:25:59.000000000 +0800
-@@ -203,7 +203,7 @@ static char *get_optstr(char *buf, char 
+diff --git a/configure.in b/configure.in
+index 668e080..4d63a43 100644
+--- a/configure.in
```

Comment:
instead of patching the tarball downloaded from `http://download.ceph.com/qa/ffsb.tar.bz2`, how about applying this patch and repackage it?

Reply:
Sounds good to me. 
BTW, how to upload the package ?

Reply:
might want to ping David Galloway on this.

Reply:
Sure.

Reply:
@tchaikov I'm wary of recommending that. It won't be obvious to someone that the ffsb code is modified (there's no git history in the tarball). I've routinely asked that patches be maintained in `qa/` even if it's ugly.

Reply:
@batrick agreed. how about keeping a copy by forking it in ceph/  ? BTW, i was packing the ffsb rpm packages. and the deb package should be trivial as there is only a single executable generated from the ffsb source tree.

Reply:
A fork on ceph/ would be acceptable and then note the tarball comes from github (it still needs hosted on ceph.com to avoid overloading github). We'll need help from a ceph org admin to clone the repo.

Reply:
@lxbsz github.com/ceph/ffsb is available for hacking now. Thanks to @tchaikov

Reply:
Cool, will update the PR.

Reply:
@batrick @tchaikov 

There is no ffsb under `https://github.com/ceph/`, did I miss something important ?

Reply:
Sorry, I can clone it from `https://github.com/ceph/` now, but still cannot find it from the web.

Reply:
Interesting... must be firewalled.

Reply:
It's okay now, not sure why it delayed for me.

Have raised one PR in ffsb to update it: https://github.com/ceph/ffsb/pull/1

Please review @tchaikov @batrick